src/boot/bootctl-status.c

```diff
@@ -415,96 +416,219 @@ int verb_status(int argc, char *argv[], void *userdata) {
+                                        print_yes_no_line(i == 0, FLAGS_SET(stub_features, stub_flags[i].flag), stub_flags[i].name);
+                        }
+                        if (!sd_id128_is_null(loader_part_uuid))
+                                printf("          ESP: /dev/disk/by-partuuid/" SD_ID128_UUID_FORMAT_STR "\n",
+                                       SD_ID128_FORMAT_VAL(loader_part_uuid));
+                        else
+                                printf("          ESP: n/a\n");
+                        printf("         File: %s%s\n", special_glyph(SPECIAL_GLYPH_TREE_RIGHT), strna(loader_path));
+                        printf("\n");
 
-                if (stub) {
-                        printf("         Stub: %s\n", stub);
-                        for (size_t i = 0; i < ELEMENTSOF(stub_flags); i++)
-                                print_yes_no_line(i == 0, FLAGS_SET(stub_features, stub_flags[i].flag), stub_flags[i].name);
-                }
-                if (!sd_id128_is_null(loader_part_uuid))
-                        printf("          ESP: /dev/disk/by-partuuid/" SD_ID128_UUID_FORMAT_STR "\n",
-                               SD_ID128_FORMAT_VAL(loader_part_uuid));
-                else
-                        printf("          ESP: n/a\n");
-                printf("         File: %s%s\n", special_glyph(SPECIAL_GLYPH_TREE_RIGHT), strna(loader_path));
-                printf("\n");
+                        printf("%sRandom Seed:%s\n", ansi_underline(), ansi_normal());
+                        have = access(EFIVAR_PATH(EFI_LOADER_VARIABLE(LoaderSystemToken)), F_OK) >= 0;
+                        printf(" System Token: %s\n", have ? "set" : "not set");
 
-                printf("%sRandom Seed:%s\n", ansi_underline(), ansi_normal());
-                have = access(EFIVAR_PATH(EFI_LOADER_VARIABLE(LoaderSystemToken)), F_OK) >= 0;
-                printf(" System Token: %s\n", have ? "set" : "not set");
+                        if (arg_esp_path) {
+                                _cleanup_free_ char *p = NULL;
 
-                if (arg_esp_path) {
-                        _cleanup_free_ char *p = NULL;
+                                p = path_join(arg_esp_path, "/loader/random-seed");
+                                if (!p)
+                                        return log_oom();
 
-                        p = path_join(arg_esp_path, "/loader/random-seed");
-                        if (!p)
-                                return log_oom();
+                                have = access(p, F_OK) >= 0;
+                                printf("       Exists: %s\n", yes_no(have));
+                        }
 
-                        have = access(p, F_OK) >= 0;
-                        printf("       Exists: %s\n", yes_no(have));
+                        printf("\n");
                 }
-
-                printf("\n");
         } else
                 printf("%sSystem:%s\n"
```

Comment:
All the output needs to be in json, including this and everything else that the verb prints

Reply:
I am still working on this.

Reply:
I completed this.